system/k3s/doinst.sh

```diff
@@ -14,5 +14,10 @@ config() {
 config etc/rancher/k3s/k3s.service.env.new
 
 if [ -x /etc/rc.d/rc.k3s ]; then
-    /etc/rc.d/rc.k3s restart > /dev/null
+  if [ ! -f /etc/rancher/k3s/k3s.yaml ]; then
+    k3s server \
+      --write-kubeconfig /etc/rancher/k3s/k3s.yaml \
+      --write-kubeconfig-mode 644 > /dev/null 2>&1 &
+  fi
+  /etc/rc.d/rc.k3s restart > /dev/null
```

Comment:
You should not restart services in doinst.sh.

Reply:
I'll remove it, but I should still keep initiate for kube